packages/nimble-components/src/dialog/tests/dialog.stories.ts

```diff
@@ -125,6 +127,9 @@ const metadata: Meta<DialogArgs> = {
         preventDismiss: {
             name: 'prevent-dismiss'
         },
+        largeMode: {
+            name: 'large-mode'
```

Comment:
Let's add a description to this too. Something like this:
```suggestion
            name: 'large-mode'
            description: 'Add the `large-mode` attribute to the dialog to display it at a larger size for experiences like a wizard or configuration dialog.'
```